packages/kolibri-zip/src/fileUtils.js

```diff
@@ -127,4 +223,5 @@ export const defaultFilePathMappers = {
   htm: DOMMapper,
   xhtml: DOMMapper,
   xml: DOMMapper,
+  bloom: BloomMapper,
```

Comment:
Please reference back to my comments on your previous PR - you have defined a mapper for a bloom filetype, but that is not a filetype that is found inside the archive file.

Instead, this work should be done to enhance the DOMMapper class.